package/MDAnalysis/analysis/reflectivity.py

```diff
@@ -0,0 +1,293 @@
+  P    31    1           0.5       5.1300       0.000    0.2000       0.
+  S    32    0.9502      0.0       2.8040       0.000    0.0000       0.
+  S    33    0.0075      1.5       4.7400       0.000    1.5000       0.
+  S    34    0.0421      0.0       3.4800       0.000    0.0000       0.
+  S    36    0.0002      0.0       3.0000       0.000    0.0000       0.
+  Cl   35    0.7577      1.5       11.650       0.000    6.1000       0.
+  Cl   37    0.2423      1.5       3.0800       0.000    0.1000       0.
+  Ar   36    0.00337     0.0       24.900       0.000    0.0000       0.
+  Ar   38    0.00063     0.0       3.5000       0.000    0.0000       0.
+  Ar   40    0.99600     0.0       1.8300       0.000    0.0000       0.
+  K    39    0.93258     1.5       3.7400       0.000    1.4000       0.
+  K    40    0.00012    -4.0       3.0000       0.000    0.0000       0.
+  K    41    0.06730     1.5       2.6900       0.000    1.5000       0.
+  Ca   40    0.96941     1.5       4.8000       0.000    0.0000       0.
+  Ca   42    0.00647     0.0       3.3600       0.000    0.0000       0.
+  Ca   43    0.00135     3.5      -1.5600       0.000    0.0000       0.
+  Ca   44    0.02086     0.0       1.4200       0.000    0.0000       0.
+  Ca   46    0.00004     0.0       3.6000       0.000    0.0000       0.
+  Ca   48    0.00187     0.0       0.3900       0.000    0.0000       0.
+"""
+
+
+def guess_isotopes(u):
+    """Add isotope topology attribute to the atoms in a universe and make
+    guesses about what the isotopic composition of each atom should be.
+
+    Parameters
+    ----------
+    u : MDAnalysis.Universe
+        A universe instance to populate with isotope attributes
+
+    Remarks
+    -------
+    After guessing, it is highly recommended to check the guesses and fix
+    any errors. For specific isotopes of hydrogen, the following formula
+    symbols should be used:
+
+    - Pr = protium
+    - D = deuterium
+    - T = tritium
+    
+    Where H is used, the natural isotopic composition will be assumed.
+    Example:
+        CH4 = methane
+        CH3D = methane with natural isotopic hydrogen except for one
+        which is deuterium
+        CPr4 = methane with no deuterium (all protium)
+
+    """
+
+    if 'isotopes' not in dir(u.atoms):
+        u.add_TopologyAttr(Isotopes(np.empty(len(u.atoms), dtype='S10')))
+
+    for atom in u.atoms:
+        try:
+            atom.isotope = type2isotope_dict[atom.type]
+        except KeyError:
+            try:
+                atom.isotope = guessers.guess_atom_element(atom.type)
+            except AttributeError:
+                atom.isotope = guessers.guess_atom_element(atom.name)
+
+
+def parse_isotope(formula):
+    """Convert a formula in the style C2H6O into a dictionary whose keys
+    are elements and values are the number of each element.
+    """
+    letters = []
+    digits = []
+    element_num_dict = {}
```

Comment:
You can do this with a `defaultdict` to avoid the try/except below..
```python
In [1]: from collections import defaultdict

In [2]: a = defaultdict(int)

In [3]: a['this'] += 2

In [4]: a['this']
Out[4]: 2
```